src/index.js

```diff
@@ -1 +1,25 @@
-import './styles/styles.css';
\ No newline at end of file
+import './styles/styles.css';
+
+import getData from './module/data.js';
+
+const card = document.querySelector('.list-items .card');
+getData().then((data) => {
+  data.meals.forEach((item) => {
+    card.innerHTML += `
+    <li>
+          <div class="card-image">
+            <img src="${item.strMealThumb}" alt="">
```

Comment:
- [ ] Develop an Alternative Text to address Accessibility and SEO.

Reply:
Amazing work @Bateyjosue !

Please can you give each meal card (`li` tag) an ID that will reflect the id of the meal ("idMeal")!
This easy selection of meal ID.